MyWebPage/index.html

```diff
@@ -33,6 +33,39 @@ <h2>Play Billar</h2>
lgAs7lV5jyJJ9hXe2SJ+Ge4UUuuQBtmGZgu/PfajuHVLi5luIYWSoMMBsDlaNKzXb+4VsIqwVuOJM7ZAHA2g+/L2hNm1RoCdkrRa/OsKhPPcgASdtiaJce7TW8MTbTx3QRKaws6yzR0MwNfSsknaM2Ldy1aUh7kAvOiqAR4B/Prv+wGwaS2b4iDOvM8ieuutHW8gn1RN+GyuGuHM7eNgf5mlvF1PM16B2Ow6XC+YiCGidYzbgeWg9hWQwmDa6ouakliGHU/s7Vv+CYNLFoyDngaawM06T1j7ikubx72UuPyO/QBfDhXZJ0ztB6eI+8aD5VauhFAKhoOkkydNCdetQ4m0qsxJliZgDz+lOu3c5DGN5K/ST8yKwcm3+jfqkv2XL2KRUULtqZPPbfpXn3EMWcVfNw6JACDfKJyrPqxk+tartDZAw90hoKr1kGSBlB5TMfOsRhrYLJ48kuozGQFk/ESNgD9jV8aVWTPdF/B4Qm+guWyqqxZjlYCEBdgQRH5alx2KFxxmY5AqqOW2rsfNnLP/u8qIvaJsM63S5Iykh8wbWGI5zlJn/VQRLW4J10j5x/X6VcU9szk1pGk4bYgZraeR1UEkaaxtVrC9nWuXBcGUACGRlJEEZYBUidy3IzUeAtlLaFsiktEA+KYkZo/mg6HmD1r0Dglhe7UkzKyPKdq0jkxn2TpE/COHJZti2CX0ks0EkncmBHl5CBRC3hAwOkSCPes/wAd7V2sG5ttbd3ChhBCqZ0Gp1PrHOsTxnt1ibwyrFtdiqyCd9GJOojl60nY4x+m+u4+zg2ZLlwaKANCWgqpMKJ6AUK432ttXQhRGlNVJAmYid68wS5cdz4lJ10JAn/dsfepX4g22TYaxP8ASpHVKkaG3jSbjXGksfhnl6H5118c43H3j+9Zu1xAk5YJJ9qv2Lh2kjy3mqwS0wl37MDJB8qoYm6xME6dJ061LbeeQ+WlSOFOkR6iadISdAu9cXaeUaVTRmB5kT0H9aL3bB5ZT6VWfDgCSD6xFFFdis76gkz5edVrl2Zj132qfEJymqrIR6c6CkMtLr1mpMhJpyIANBr571We64J5em1Ayw9g6TO3lS7sdfof6U20WI1U+tdKt5+1KwLnckbx7U7XmBVM4m4RObbyFQh2JjMftQBeN5V3iddahOOPT6/2qPuydMxPzpxwfhzZhQkJtE9q4TBn3irP4kA6jeqysBsIHLWTTgk7/wBtdtqolhnDIpAYDM0EHaYkGR9qlGL7sw8bEAchQnDW2U6NGvImrhX4QzTp9vvVxdIxnG2FG4mlxArQIJhjGkwNOceGfnQjH5UZROYCTuSJHr60SsCZmNQOk1Q7UYpRlRYzsAW28I2jXaft60SdrJMFUqRlcTdd2LPqxJJPWTr6CiXDLfhJKkySOkFYJAbrBqHD2Fff4dNjDEEkTsRGkdZIjSaL4aQIJJbOz6AakhVHOCYG8f3yjbwdM2ki9w+81jxKFJI3aQNd9Awk+9XT2muDQhCOgBE+mpoNfxrRsNBrBkbzr++VVWcc1+tKXHH4EOSVbC+K4yH2Q+/9qbZ4oArL3ZzNzzdOW1CDdEHw06ziJI8Omuv9/WkuOOhvklsl4zinuWikALIJ1k6aj9Ki4VwZrq/GqCJGb8xEnwgjX7dabirhiDMGI69aMcKuh10Lhm8IUE5fH4WJAGpg7DqDU8txrqXxtP8AIG31Nsm3bUIoJlVZ2UEwYliSY2361WtW3YwATO4iTRrH4ZxbJaSM5blu0hiY5ytC0RlM7eIcvv5f3rVxa2ZKcXlBvCW0NsISvwM0gHMmSIUz/N9M1VeE8SxVu4tvDHMXaFRoKMenijL6giosfda3cOgkqBoZBggDXnqn0qx2bv8Ad30usYCEmBIJlSsDy1o0grtI9H47wO3jLSJfGV1Ah0jMhMFwpYGUJGx8q8Z4rgLuGuNZujK415QwkgOI5GJr1Ru0rOZUAfU+9ZjtVN4O1xS/8M5G0hCkuT5Tt9KyUsnT48WYe3d6/pVs42dG1HXboOhjQUOQcvn/AFrublNaGJexT28qm2rBs2rHQEZdhHnP0ptq8QAxPM+YqqlyBEnf7dKsvcBXKZknc6mN/nyoAvW8cAddiOVStj1jf0kGhli0XIUe/lWjfhlooSNDl01PTQxRRLoDPxAc/wBantYmRIOnSR9qp8UsW1mGbN/KR58yKoWbQYxMff5U0w6oNuwI1APnH61XCDoakS3ppUj2z+/emIpsoPP3oddckxyoo1s5T4Qf350OxEA6fakykRh2TUUQscQ0139P70NYnnt9KQpFhBLY9QdCP396rukGu0qCSZDA8X796kCaEzIG31iKVKmhMntZSNJnodZ05GN6dZltAYiNP7xSpVZmWzYaZM7aGYEbbfKnfh9ZGvrv/alSpC9E+Jvd3bzswMCAok67Lr96z1i01xySMxJJPOYgmR/LB18qVKpkXxpHoPBeB2Ew11rqhmZCwdjrmUghV125E8yB0AGUxhtqVNsNrurbL6Hc/OlSrSSpGEJNt2Msss6r8uQmrK2wQCAIkHXf/qlSqfZqxzWlP5R1j98q7Y4aC4KrI6TzjQwNYmlSpx2Z8kmolHijgXsrDwpoJ9ATM7amiXD8Xaa2bZCqwIKncArmlTyhp+mtcpVM/wAhx/FFvFXS1tU11MmCJJGg20I/pVDEWAuWD4jJ9vKlSrV6Mo7OfhcwzkyTO0etLDW4aB8xrXaVYyOiDNNw3CTFEsZw0d26sYDKRMajMMu3PelSrFnof1PMMZw69hi/eW2ysGRXg5CCdw20kA6HWDO1CylKlWhz0dYcqmzScp6QP7UqVMR1MU1tpHSOu/z3o3hmbQi4rgiYyxymNDoaVKgiQP4gjXGGniPQiDB1qggyPDjXkOQ9aVKgaC+DYgjNoOoqy9wTA2/ce9KlVIhlQvrB1FVcSgYQOvL99KVKkxooXlM+XL0pIdKVKkaH/9k=" alt="call_of_duty_warzone">
```

Comment:
Please fix in the same way of the main comment to all the images. Including the profile image